IFC_oM/Enums/Discipline.cs

```diff
@@ -0,0 +1,48 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.ComponentModel;
+
+namespace BH.oM.Adapters.IFC
+{
+    /***************************************************/
+
+    [Description("Enumerator allowing choosing to which discipline (and corresponding namespace) should IFC elements be converted on pull.")]
+    public enum Discipline
+    {
+        [Description("Default discipline to be used.")]
```

Comment:
```suggestion
        [Description("Default discipline. Depending on the type, the most suitable discipline will be used per each object.")]
```

As per our conversation yesterday. This helps clarifying that if Discipline is left to `Undefined`, the actual discipline is chosen differently per every object.

Reply:
Done